test/arkane/encorr/isodesmicTest.py

```diff
@@ -278,6 +275,8 @@ def test_creating_error_canceling_schemes(self):
         scheme = ErrorCancelingScheme(
             self.propene,
             [self.butane, self.benzene, self.caffeine, self.ethyne],
+            "rc2",
+            True,
             True,
             True,
```

Comment:
probably using kwargs to improve readability.

Reply:
Thanks, done